lib/pages/home/pages/menu/pages/user_invites/user_invites.dart

```diff
@@ -203,8 +211,8 @@ class OBUserInvitesPageState extends State<OBUserInvitesPage> {
     try {
       await Future.wait([
         _refreshUser(),
-        _acceptedInvitesGroupController.refresh(),
-        _pendingInvitesGroupController.refresh()
+        _hasAcceptedInvites ?_acceptedInvitesGroupController.refresh() : _refreshAcceptedInvites(),
+        _hasPendingInvites ? _pendingInvitesGroupController.refresh() : _refreshPendingInvites(),
       ]);
```

Comment:
if `_hasAcceptedInvites` is false, we hide the widget so the corresponding controllers have no state and their refresh methods return. So we explictly call the `_refresh` methods. 
This happens when the first invite is created before the Pending/Accepted sections are rendered and hence there is no state in the controllers.